WordPressShared/WordPressShared/Core/Views/WPNoResultsView.m

```diff
@@ -136,6 +156,19 @@ - (void)layoutSubviews {
     }
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
+{
+    [super traitCollectionDidChange:previousTraitCollection];
```

Comment:
I'm just curious, but if I recall correctly, this would also be triggered on orientation changes, so would still be necessary to observe `UIDeviceOrientationDidChangeNotification` ?

Reply:
Good catch!